### PR TITLE
Change parameters' form to hex string vector

### LINKED ISSUE #384: Change serialization of Bytes to string
Currently, the type of `parameters` field in `Asset` and `AssetTransferOutput` is `Vec<Bytes>`.

The JSON serialization of `Bytes` is `Uint8Array` and it's hard to read(or debug) in the SDK environment.

image

### PR DESCRIPTION
**Pull request by HoOngEe**
change types of the field "parameters" in AssetTransferOutput and
AssetMintOutput from Vec<Bytes> to Vec<String> for the sake of
legibility.

It closes #384.

### CODE REVIEW

File: rpc/src/v1/types/asset_output.rs [hunk 1]
```diff
@@ -13,16 +13,20 @@
...
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
+extern crate rustc_serialize;
 
 use cjson::uint::Uint;
 use ctypes::transaction::{AssetMintOutput as AssetMintOutputType, AssetTransferOutput as AssetTransferOutputType};
-use primitives::{Bytes, H160, H256};
+use primitives::{H160, H256};
+use rustc_serialize::hex::{FromHex, FromHexError, ToHex};
+
+use std::iter::FromIterator;
```
Comment (sgkim126) [inline rpc/src/v1/types/asset_output.rs]: Please move this line before line 18.
https://github.com/CodeChain-io/codechain/wiki/Coding-Style#imports-order

File: rpc/src/v1/errors.rs [hunk 2]
```diff
@@ -67,6 +86,18 @@ pub fn core<T: Into<CoreError>>(error: T) -> Error {
     }
 }
 
+pub fn conversion<T: Into<ConversionError>>(error: T) -> Error {
+    let error = error.into();
+    match error {
+        ConversionError::Key(error) => transaction_core(CoreError::Key(error)),
+        ConversionError::Hex(error) => Error {
+            code: ErrorCode::ServerError(codes::HEX_ERROR),
+            message: format!("{}", error),
```
Comment (sgkim126) [inline rpc/src/v1/errors.rs]:
```rust
message: error.to_string()
```

File: rpc/src/v1/types/asset_output.rs (hunk 1 above)
Comment (HoOngEe) [reply rpc/src/v1/types/asset_output.rs]: Fixed.

File: rpc/src/v1/errors.rs (hunk 2 above)
Comment (HoOngEe) [reply rpc/src/v1/errors.rs]: Fixed.

File: rpc/src/v1/types/action.rs [hunk 3]
```diff
@@ -15,11 +15,15 @@
...
 use cjson::uint::Uint;
-use ckey::{Error as KeyError, NetworkId, PlatformAddress, Public, Signature};
-use ctypes::transaction::Action as ActionType;
+use ckey::{NetworkId, PlatformAddress, Public, Signature};
+use ctypes::transaction::{Action as ActionType, AssetMintOutput as AssetMintOutputType};
 use ctypes::ShardId;
 use primitives::{Bytes, H160, H256};
 
+use rustc_serialize::hex::FromHexError;
+use std::iter::FromIterator;
```
Comment (sgkim126) [inline rpc/src/v1/types/action.rs]: Put it on line 16.

File: rpc/src/v1/types/unsigned_transaction.rs [hunk 4]
```diff
@@ -15,9 +15,12 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 use cjson::uint::Uint;
-use ckey::{Error as KeyError, NetworkId};
+use ckey::NetworkId;
 use ctypes::transaction::IncompleteTransaction;
```
Comment (sgkim126) [inline rpc/src/v1/types/unsigned_transaction.rs]: Remove this blank line.

File: rpc/src/v1/types/action.rs [hunk 5]
```diff
@@ -15,11 +15,15 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 use cjson::uint::Uint;
-use ckey::{Error as KeyError, NetworkId, PlatformAddress, Public, Signature};
-use ctypes::transaction::Action as ActionType;
+use ckey::{NetworkId, PlatformAddress, Public, Signature};
+use ctypes::transaction::{Action as ActionType, AssetMintOutput as AssetMintOutputType};
 use ctypes::ShardId;
 use primitives::{Bytes, H160, H256};
```
Comment (sgkim126) [inline rpc/src/v1/types/action.rs]: Remove this blank line.

**Comment by sgkim126** [comment]
@HoOngEe Please change the sdk and fix integration tests.

**Pull request closed without merging**